src/sentry/incidents/logic.py

```diff
@@ -995,6 +999,30 @@ def get_triggers_for_alert_rule(alert_rule):
     return AlertRuleTrigger.objects.filter(alert_rule=alert_rule)
 
 
+def trigger_incident_triggers(incident):
+    from sentry.incidents.tasks import handle_trigger_action
+
+    triggers = IncidentTrigger.objects.filter(incident=incident).select_related(
+        "alert_rule_trigger"
+    )
+    for trigger in triggers:
+        if trigger.status == TriggerStatus.ACTIVE.value:
+            with transaction.atomic():
+                method = "fire" if trigger.status == TriggerStatus.ACTIVE.value else "resolve"
```

Comment:
We only need `resolve` here, since we're only ever going to do this when `trigger.status == TriggerStatus.ACTIVE.value`